### DISCUSSION TITLE
Ghost博客引入前端出错

### DISCUSSION THREAD

**Discussion by ShadowsSide** [ArtalkJS/Artalk#230] [Q&A] [answered]
使用通用代码引入，可以显示评论区，但是无法拉取评论
![image](https://user-images.githubusercontent.com/61958350/182083065-3a207016-7b6e-410b-8450-61adabfa95a5.jpeg)

**Comment by qwqcode** [comment]
可以右键检查，提供一下 Console 里面具体的报错信息吗？

也许是跨域之类的问题

**Comment by ShadowsSide** [comment]
> 可以右键检查，提供一下 Console 里面具体的报错信息吗？
> 
> 
> 
> 也许是跨域之类的问题

好的，我9点到家查看一下，到时截图上来，谢谢。

**Comment by ShadowsSide** [comment]
![image](https://user-images.githubusercontent.com/61958350/182159509-eb7b0ec7-8dde-4da5-9efc-dabe1f30f511.png)

**Comment by qwqcode** [comment, marked as answer]
你的网站是 https 协议，Artalk 的后端程序 API 需要相同的协议才行，然而现在是 http 协议

**Reply by ShadowsSide** [reply]
好的,我尝试在后端文件中的SSL配置下.谢谢

**Reply by gfjz** [reply]
我也是这种情况，配置反代需要注意什么吗，我artalk后端也是https，我的测试页面是：https://blog.gfjzz.cn/about
